R/client-cloud.R

```diff
@@ -161,7 +161,7 @@ cloudClient <- function(service, authInfo) {
       GET(service, authInfo, path, query)
     },
 
-    createApplication = function(name, title, template, accountId, appMode) {
+    createApplication = function(name, title, template, accountId, appMode, contentCategory) {
```

Comment:
```suggestion
    createApplication = function(name, title, template, accountId, appMode, contentCategory = NULL) {
```

? (Would simplify a couple of the tests)